### PR TITLE
Prepare Rules page tables

### LINKED ISSUE #330: Prepare Rules page tables
Ref https://github.com/stylelint/stylelint/pull/6960#pullrequestreview-1495120478

### PR DESCRIPTION
**Pull request by ybiquitous**
> Which issue, if any, is this issue related to?

Closes #330
See stylelint/stylelint#6960

> Is there anything in the PR that needs further explanation?

The recommended config will be removed in the next release, so please merge this pull request then.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -100,7 +100,7 @@
     "remark": "^14.0.3",
     "remark-cli": "^11.0.0",
     "remark-gfm": "^3.0.1",
-    "stylelint": "https://github.com/stylelint/stylelint/tarball/15.9.0",
+    "stylelint": "https://github.com/stylelint/stylelint/tarball/HEAD",
```
Comment (ybiquitous) [inline package.json:103]: - [ ] Change to the next version

**Comment by ybiquitous** [comment]
Preview: https://deploy-preview-331--stylelint.netlify.app/user-guide/rules/

**Review by jeddy3 — APPROVED** [review]
Thanks for doing this!

The page looks cleaner.

File: package.json (hunk 1 above)
Comment (mattxwang) [reply package.json:103]: Changed this in #333!

**Pull request merged**